bagel/models.py

```diff
@@ -63,8 +73,11 @@ class PhenotypicSession(Session):
 
 
 class ImagingSession(Session):
+    # NOTE: imaging session have at least one acquisition OR at least one completed pipeline to be valid?
     hasFilePath: Optional[str] = None
-    hasAcquisition: List[Acquisition]
+    # TODO: change name to hasAcquisitions?
```

Comment:
I disagree here. It looks like one attribute can contain many things here in pydantic, but in the graph we will have one edge called (hasAcquisition) for each Acquisiton.